Sorts/QuickSortRecursive.js

```diff
@@ -43,23 +58,55 @@ const quickSort = (inputList, low, high) => {
  * @returns {number} `pIndex` pivot index value.
  */
 const partition = (partitionList, low, high) => {
-  const pivot = partitionList[high]
+  const mid = Math.floor((low + high) / 2)
+  // get the median of three is good technique for partitioning to be sure that the 2 sub-arrays will be almost equal or nearly equal in size
+  const pivot = medianOfThree(partitionList, low, mid, high) // Find the pivot element
+
+  // Move pivot to the end
+  let pivotIndex = partitionList.indexOf(pivot)
+  ;[partitionList[pivotIndex], partitionList[high]] = [
```

Comment:
I would rename some variables to make this less verbose.

Reply:
ok that will be better